src/lib/stepper/step-header.html

```diff
@@ -1,6 +1,5 @@
 <div class="mat-step-header-ripple" mat-ripple [matRippleTrigger]="_getHostElement()"></div>
-<div [class.mat-step-icon]="state !== 'number' || selected"
-     [class.mat-step-icon-not-touched]="state == 'number' && !selected"
+<div class="mat-step-icon-{{selected ? 'selected' : 'not-selected'}} mat-step-icon-state-{{state}} mat-step-icon"
```

Comment:
can we just use `.mat-icon:not(.mat-step-icon-selected)` instead of having both classes?

Reply:
sounds good!

Reply:
done!